src/hotspot/cpu/zero/stack_zero.cpp

```diff
@@ -26,11 +26,10 @@
 #include "precompiled.hpp"
 #include "interpreter/interpreterRuntime.hpp"
 #include "interpreter/zero/bytecodeInterpreter.hpp"
-#include "runtime/javaThread.hpp"
-#include "stack_zero.hpp"
-#include "stack_zero.inline.hpp"
 #include "runtime/frame.inline.hpp"
 #include "runtime/javaThread.inline.hpp"
+#include "stack_zero.hpp"
+#include "stack_zero.inline.hpp"
```

Comment:
Existing, but it is redundant to include the .hpp and the .inline.hpp file as the later always includes the former.

Reply:
You're right.  I touched that line but didn't even see it.  I confirmed stack_zero.inline.hpp includes stack_zero.hpp, so removed the latter from stack_zero.cpp.  We'll let GHA confirm.